runtime/j9vm/jvm.c

```diff
@@ -1297,7 +1322,7 @@ preloadLibraries(void)
 	const char *origLibpath = NULL;
 #endif /* defined(AIXPPC) */
 
-	if (j9vm_dllHandle != 0) {
+	if (0 != j9vm_dllHandle) {
```

Comment:
`j9vm_dllHandle` is a pointer which should be compared with `NULL` instead of `0`.